src/concurrency/timestamp_ordering_transaction_manager.cpp

```diff
@@ -637,8 +637,8 @@ void TimestampOrderingTransactionManager::PerformDelete(
 
   auto transaction_id = current_txn->GetTransactionId();
 
-  PL_ASSERT(GetLastReaderCommitId(tile_group_header, old_location.offset) ==
-            current_txn->GetCommitId());
+  // PL_ASSERT(GetLastReaderCommitId(tile_group_header, old_location.offset) ==
```

Comment:
This assertion does not pass when modifying another table in the executor.

Reply:
Ok. We shouldn't leave a commented out assertion here though, without an explanation. Do we understand if

- The assertion is wrong?
- The assertion is correct, but there is a bug, in the new code or existing code?

One or more of the following are suggested:

- Add a comment explaining why the assertion is commented out
- File an issue about it
- Understand what is going on, and fix or remove the assertion accordingly.